assets/model_monitoring/components/src/data_drift_compute_metrics/numerical_data_drift_metrics.py

```diff
@@ -6,7 +6,7 @@
 import pandas as pd
 import pyspark.sql as pyspark_sql
 import pyspark.sql.functions as F
-from scipy.stats import ks_2samp
+from scipy.stats import ks_2samp, wasserstein_distance
 from synapse.ml.exploratory import DistributionBalanceMeasure
```

Comment:
You no longer need to import DistributionBalanceMeasure, as it is no longer being used.

Reply:
This is still being used by JensenShannonMetrics

Reply:
Ok, let's keep it until we substitute the Jensen-Shannon Distance as well.

Reply:
Replaced the JensenShannonDistance calculation logic too, to mitigate the entropy function issue (https://github.com/evidentlyai/evidently/issues/494).

Reply:
So I removed the line to import DistributionBalanceMeasure. The missing module error should be gone.